src/power_grid_model_io_native/_core/power_grid_model_io_core.py

```diff
@@ -146,15 +146,15 @@ def error_message(self) -> str:  # type: ignore[empty-body]
         pass  # pragma: no cover
 
     @make_c_binding
-    def create_vnf_converter(self, data: str, experim_feature: int) -> PgmVnfConverterPtr:  # type: ignore[empty-body]
+    def create_pgm_vnf_converter(self, data: str, exp_feature: int) -> PgmVnfConverterPtr:  # type: ignore[empty-body]
```

Comment:
just call write it out:
```suggestion
    def create_pgm_vnf_converter(self, data: str, experimental_features: int) -> PgmVnfConverterPtr:  # type: ignore[empty-body]
```

Reply:
The main issue I had with this line was that it was too long 😄 since it was too long it was being reformatted and split into three lines. The issue then became that the name of the function "create_pgm_vnf_converter" wasn't on the same line as the comment # type: ignore[empty-body]. This resulted in an error saying that the function does not have a return.
I tried splitting the line myself in two ways, but it was still being reformatted.
The simplest solution was to place the comment next to the name of the function, but now it looks quite ugly.

Reply:
what about something of the order of this? please also have a look at the equivalent file in the PGM. We've got the same issue there so there should already be a solution for this

```suggestion
    def create_pgm_vnf_converter(  # type: ignore[empty-body]
        self, data: str, experimental_features: int
    ) -> PgmVnfConverterPtr:
```

Reply:
Yep, this is the current implementation. In PGM I saw that there are cases exactly like that and the solution is exactly this.